packages/ttd.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,24 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    $toolName = "ttd"
+    $category = "Debuggers"
+
+    # From https://aka.ms/ttd/download
+    $bundleUrl = "https://windbg.download.prss.microsoft.com/dbazure/prod/1-11-319-0/TTD.msixbundle"
+    $bundleSha256 = "f7b80731c3a6994b3763c4100073b101965327d6556fa4bfb553d70ce49be366"
+    $bundlePath = Join-Path ${Env:TEMP} "$($toolName)_1-11-319-0.msixbundle"
```

Comment:
I don't think it's necessary to specify the version here. `Join-Path ${Env:TEMP} "$toolName.msixbundle"` is enough.

Reply:
I agree, it will need less changes when updating

Reply:
@naacbin Done.